docs/src/bootstrap.md

```diff
@@ -275,6 +275,39 @@ spec:
     as queries are run as a superuser and can disrupt the entire cluster.
     An error in any of those queries interrupts the bootstrap phase, leaving the cluster incomplete.
 
+Moreover, you can specify a list of Secrets and/or ConfigMaps which contains SQL script that will be executed after the database is created and configured. These SQL script will be executed using the **superuser** role (`postgres`), connected to the database specified in the `initdb` section:
+
+```yaml
+apiVersion: postgresql.cnpg.io/v1
+kind: Cluster
+metadata:
+  name: cluster-example-initdb
+spec:
+  instances: 3
+
+  bootstrap:
+    initdb:
+      database: app
+      owner: app
+      postInitApplicationSQLRefs:
+        secretRefs:
+        - name: my-secret
+          key: secret.sql
+        configMapRefs:
+        - name: my-configmap
+          key: configmap.sql
+  storage:
+    size: 1Gi
+```
+
+!!! Note
+    The SQL scripts referenced in `secretRefs` will be executed before the ones referenced in `configMapRefs`. For both sections the SQL scripts will be executed respecting the order in the list.
```

Comment:
Is this still true?
now we do sort.Strings(files)

`	
if err := fileutils.EnsureDirectoryExist(info.PostInitApplicationSQLRefsFolder); err != nil {
		return fmt.Errorf("could not find directory: %s, err: %w", info.PostInitApplicationSQLRefsFolder, err)
	}

	files, err := fileutils.GetDirectoryContent(info.PostInitApplicationSQLRefsFolder)
	if err != nil {
		return fmt.Errorf("could not get directory content from: %s, err: %w",
			info.PostInitApplicationSQLRefsFolder, err)
	}

	sort.Strings(files)

	for _, file := range files {
		sql, ioErr := fileutils.ReadFile(path.Join(info.PostInitApplicationSQLRefsFolder, file))
		if ioErr != nil {
			return fmt.Errorf("could not read file: %s, err; %w", file, err)
		}

		if err = info.executeQueries(sqlUser, []string{string(sql)}); err != nil {
			return fmt.Errorf("could not execute queries: %w", err)
		}
	}
`